MetamodelUML.md

```diff
@@ -839,22 +857,42 @@ View packages worden naar de volgende aspecten gespecificeerd, analoog aan
+| van toepassing op **Enumeratie**                | 0..1              | Bindinding aan een Enumeratie                                                     |                                       |    |                              |             |
+| van toepassing op **Enumeratiewaarde**          | 0..1              | Bindinding aan een Enumeratiewaarde                                               |                                       |    |                              |             |
+| van toepassing op **Referentielijst**           | 0..1              | Bindinding aan een Referentielijst                                                |                                       |    |                              |             |
+| van toepassing op **Referentie-element**        | 0..1              | Bindinding aan een Referentie-element                                             |                                       |    |                              |             |
+| van toepassing op **Primitief datatype**        | 0..1              | Bindinding aan een Primitief datatype                                             |                                       |    |                              |             |
+| van toepassing op **Gestructureerd datatype**   | 0..1              | Bindinding aan een Gestructureerd datatype                                        |                                       |    |                              |             |
+| van toepassing op **Data-element**              | 0..1              | Bindinding aan een Data-element                                                   |                                       |    |                              |             |
+| van toepassing op **Keuze**                     | 0..1              | Bindinding aan een Keuze                                                          |                                       |    |                              |             |
+| van toepassing op **Domein**                    | 0..1              | Bindinding aan een Domein                                                         |                                       |    |                              |             |
+| van toepassing op **Extern**                    | 0..1              | Bindinding aan een Extern                                                         |                                       |    |                              |             |
+| van toepassing op **Informatiemodel**           | 0..1              | Bindinding aan een Informatiemodel                                                |                                       |    |                              |             |
+| van toepassing op **View**                      | 0..1              | Bindinding aan een View                                                           |                                       |    |                              |             |
```

Comment:
klopt `0..1` hier wel? En bij de andere bindingen ook. Volgens mij moet dit overal `0..*` zijn.

Verder vraag ik me af deze regels uberhaupt nodig zijn hier. De relatie in het metamodel loopt volgens mij omgekeerd. Dit lijkt dan ook dubbelop